### DISCUSSION TITLE
Publish extension to https://open-vsx.org/

### DISCUSSION THREAD

**Discussion by nojaf** [ionide/ionide-vscode-fsharp#1483] [Ideas]
Hello, I use Ionide almost exclusively with gitpod.io and it still is a bit of a hassle to install it.
As far as I understand all of it, the gitpod yml configuration supports extensions published on https://open-vsx.org/.
You can bring your own extension if you manage to get the vsix file in the container and install from there.

However, I feel like the registry is a more sustainable solution.
Now, I am aware that the extension cannot be published as is because of `vsdebug` license thing.
But could we have a separate version made without debugging support to work around this?

**Comment by Krzysztof-Cieslak** [comment]
This is already discussed in https://github.com/ionide/ionide-vscode-fsharp/discussions/1406
